Assets/Scripts/Data/SaveFileHandler.cs

```diff
@@ -0,0 +1,59 @@
+using System.IO;
+
+using UnityEngine;
+
+namespace FMS.TapperRedone.Data
+{
+    public interface ISaveFileHandler
```

Comment:
Interface, in case we eventually have to look at save files in some other location, or even on web